### ISSUE TITLE
Treating drug resistant epilepsy (DRE) 

### ISSUE THREAD

**Issue by dlee138** [openconnectome/Statistical-Connectomics-Class#252]
How does knowing the connentomics of a seizure onset zone translate into a treating the disease itself?

**Comment by DSP137** [comment]
Are we trying to determine the connectomics of an onset zone in order to know the structure or in order to pinpoint the onset zone? I think that if we are just trying to pinpoint what the onset zones are, then this could later allow us to target this region in treatments.

**Comment by aceeccc** [comment]
The characteristics of the entire network of all electrodes will show something cool. I'm not going to spoil the results :P

**Comment by aceeccc** [comment]
http://ieeexplore.ieee.org/xpls/icp.jsp?arnumber=6347012

This is a paper that comes before the one presented today, which was where they got the idea to use states I think.

**Comment by SandyaS72** [comment]
For the purposes of what the Sarma lab is trying to do, they're looking at patients for whom the only option left is really surgery at this point (DBS for epilepsy has only super recently been FDA-approved). But even so, the current failure rate of surgery is really high for certain types of epilepsy. So the point of looking at the connectomics of the ictal onset zone is to see if there's a better way to identify it based on that, because currently clinicians are literally just visually scanning hundreds of channels (they're not really thinking of a network structure in their heads).

**Comment by akim1** [comment]
There is an area of the connectomics which will go through stages of highly connected and least connected states that would be a candidate for removal.

**Comment by SandyaS72** [comment]
Yes, the presence of "foci-hot" and "foci-cold" states in Sam's paper are what inspired the idea to look for the epileptogenic zone based on network dynamics in a subsequent project.

**Comment by mblohr** [comment]
In the study, it appeared that the least connected states shortly after seizure onset was a greater indicator of SOZ than highly connected states towards the end of seizure.  Patients with higher success rates after resection surgery showed the former with high significance.

**Comment by mblohr** [comment]
Another application of the study appeared to be assisting clinicians in suggesting electrode coverage may be inadequate, in cases where AUC values were close to chance level.

**Comment by SandyaS72** [comment]
Yes the "signature" that we look for associated is primarily about the sudden decrease in centrality near onset. In most cases, mid-to-late seizure, the centrality increases again, but another big problem is that the ends of seizures are not annotated nearly as specifically as onsets. There's already some window of "wiggle room" in the actual start of seizure from the clinical annotation, and it's much worse for the clinically annotated offset. That's why it's much harder to attach a lot of weight to things that happen at the "end" of seizure... you don't really know that that was the end of seizure.

**Comment by SandyaS72** [comment]
And yes- one of the main things we're working on now is atleast being able to tell if the clinician's guess at the EZ is very likely to be unsuccessful, usually due to inadequate samplng area, so that they don't go into a super expensive and risky surgery with a high chance of failure.

**Comment by mblohr** [comment]
I was wondering why it was more difficult to attach weight to things happening towards the end of the seizure, i.e., the increased centrality feature.... that makes a lot of sense!

**Comment by mblohr** [comment]
SandyaS72 - are you all working towards a more structural-based (versus functional) network model?  Dr. Sarma also mentioned some control theory work her lab is doing in a paper on "Fragility Networks" - really interesting applications.

**Comment by akim1** [comment]
What does that mean exactly structural-based network model? Based on actual neuron connections as opposed to action potential measurements?

**Comment by mblohr** [comment]
Yes, actual neuron connections/models:  http://www.researchgate.net/publication/273784705_Closed-loop_control_of_a_fragile_network_application_to_seizure-like_dynamics_of_an_epilepsy_model

**Comment by adjordan** [comment]
I'm also in Sarma's lab. The we assume when we look at correlations between electrode reading that there is already structural and functional connectivity.

**Comment by adjordan** [comment]
Sorry, that wasn't very clear. We assume that when we calculate an adjacency matrix with the data that both structure and function are already playing a part the raw voltage recordings we're getting. So, it is often the case that when we look at different time series transforms for electrodes that are physically near each other in the brain, they have similar trends.

**Comment by adjordan** [comment]
Also, to add on to the previous comments, brain resections that are currently done really overshoot the amount of brain tissue that actually needs to be removed, leading to loss of cognitive function, memory, etc. So an algorithm that localizes the EZ would be able to help physicians remove where they need to remove and no more than they need to remove.
